allensdk/brain_observatory/behavior/data_objects/metadata/behavior_metadata/project_code.py

```diff
@@ -0,0 +1,41 @@
+from pynwb import NWBFile
+
+from allensdk.core import DataObject
+from allensdk.core import \
+    LimsReadableInterface
+from allensdk.internal.api import PostgresQueryMixin
+from allensdk.core import NwbReadableInterface
+
+
+class ProjectCode(DataObject, LimsReadableInterface, NwbReadableInterface):
+    """Unique identifier for the project this BehaviorSession is associated
+    with. Project ids can be used internally to extract a project code.
+
+    If project_Code is null/None, we set the value to string 'Not Available'.
+    """
+
+    def __init__(self, project_code: str = 'Not Available'):
```

Comment:
Another way to do it is to make this optional and set it to this default value if `None`. That will fix the duplication of `if project_code is None` checks.